src/deploymentModals.jsx

```diff
@@ -0,0 +1,213 @@
+        <Button key="cleanup-accept"
+            variant="primary"
+            isDisabled={buttonLoading || (!deleteTemporaryFiles && !deleteRPMmetadata && !deletePendingDeployments && !deleteRollbackDeployments)}
+            isLoading={buttonLoading}
+            onClick={() => doCleanup()}
+        >
+            {_("Clean up")}
+        </Button>,
+        <Button key="cleanup-cancel" variant="link" onClick={Dialogs.close}>
+            {_("Cancel")}
+        </Button>
+    ];
+
+    return (
+        <Modal isOpen
+            id="cleanup-deployment-modal"
+            title={_("Remove temporary files")}
+            position="top"
+            variant="small"
+            onClose={Dialogs.close}
+            actions={actions}
+        >
+            {error &&
+                <Alert variant="danger"
+                    isInline
+                    title={error}
+                />
+            }
+            <Stack>
+                <Checkbox label={_("Temporary files")}
+                    key="temporary-files"
+                    id="temporary-files-checkbox"
+                    onChange={(_, isChecked) => setDeleteTemporaryFiles(isChecked)}
+                    isChecked={deleteTemporaryFiles}
+                />
+                <Checkbox label={_("RPM repo metadata")}
+                    key="rpm-repo-metadata"
+                    id="rpm-repo-metadata-checkbox"
+                    onChange={(_, isChecked) => setDeleteRPMmetadata(isChecked)}
+                    isChecked={deleteRPMmetadata}
+                />
+                <Checkbox label={_("Pending deployment")}
+                    key="pending-deployment"
+                    id="pending-deployment-checkbox"
+                    onChange={(_, isChecked) => setDeletePendingDeployments(isChecked)}
+                    isChecked={deletePendingDeployments}
+                />
+                <Checkbox label={_("Rollback deployment")}
+                    key="rollback-deployment"
+                    id="rollback-deployment-checkbox"
+                    onChange={(_, isChecked) => setDeleteRollbackDeployments(isChecked)}
+                    isChecked={deleteRollbackDeployments}
+                />
+            </Stack>
+        </Modal>
+    );
+};
+
+export const ResetModal = ({ os }) => {
+    const Dialogs = useDialogs();
+
+    const [removeOverlays, setRemoveOverlays] = useState(false);
+    const [removeOverrides, setRemoveOverrides] = useState(false);
+    const [error, setError] = useState("");
+    const [buttonLoading, setButtonLoading] = useState(false);
+
+    const doReset = () => {
+        setButtonLoading(true);
+        setError("");
+
+        const resetFlags = {};
+        if (removeOverlays) {
+            // remove all overlayed packages
+            resetFlags["no-layering"] = { t: "b", v: true };
+        }
+        if (removeOverrides) {
+            // remove all overrides
+            resetFlags["no-overrides"] = { t: "b", v: true };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-403-20231128-135012-30a3544e-fedora-coreos-devel//Coverage/lcov/github-pr.diff.gcov.html)